keras_retinanet/bin/train.py

```diff
@@ -355,6 +375,8 @@ def csv_list(string):
     parser.add_argument('--no-evaluation',   help='Disable per epoch evaluation.', dest='evaluation', action='store_false')
     parser.add_argument('--freeze-backbone', help='Freeze training of backbone layers.', action='store_true')
     parser.add_argument('--random-transform', help='Randomly transform image and annotations.', action='store_true')
+    parser.add_argument('--image-min-side', help='Freeze training of backbone layers.', type=int, default=800)
```

Comment:
Can you change the name of the help description?

Reply:
For sure